scripts/MKW_SaveGhostToBothCSV.py

```diff
@@ -0,0 +1,24 @@
+from dolphin import gui
+import TTK_Lib
+
+"""
+MKW_SaveGhostToBothCSV
+
+This script takes the current ghost's inputs and write them to the ghost and player csvs
+"""
+
+def main() -> None:
+    gui.add_osd_message("Script started")
+    
+    # Convert internal RKG to input list
+    input_sequence = TTK_Lib.readFullDecodedRKGData(TTK_Lib.PlayerType.GHOST)
+    
+    if (input_sequence is None or len(input_sequence) == 0):
+        gui.add_osd_message("Not in race")
+        return
+    
+    TTK_Lib.writeToCSV(input_sequence, TTK_Lib.PlayerType.GHOST)
+    TTK_Lib.writeToCSV(input_sequence, TTK_Lib.PlayerType.PLAYER)
+
+if __name__ == '__main__':
+    main()
```

Comment:
We should add a check to see if the player is in a race to prevent crashes (applies to all relevant files).

```suggestion
if __name__ == '__main__':
    if classes.RaceInfo.stage() >= 1:
       # Convert internal RKG to input list
        main()
    else:
        gui.add_osd_message("Script did not run successfully due to the player not being in a race.")
```

Reply:
Forgot to mention this will require importing `mkw_classes` to these scripts.

Reply:
I've made changes to `readFullDecodedRKGData` to catch this scenario and show a sufficient error msg.